packages/create-subscription/src/__tests__/createSubscription-test.internal.js

```diff
@@ -352,10 +377,12 @@ describe('createSubscription', () => {
 
     ReactNoop.render(<Parent observed={observableA} />);
     expect(ReactNoop.flush()).toEqual(['Subscriber: a-0', 'Child: a-0']);
+    expect(log).toEqual(['Parent.componentDidMount']);
```

Comment:
Why are you using `log` instead of `ReactNoop.yield`?

Reply:
How would I verify that commit hadn't happened with the yeild